haskell-src/Concordium/Types/Execution.hs

```diff
@@ -945,6 +950,7 @@ getPayload spv size = S.isolate (fromIntegral size) (S.bytesRead >>= go)
                 cbTransactionFeeCommission <- maybeGet 5
                 cbBakingRewardCommission <- maybeGet 6
                 cbFinalizationRewardCommission <- maybeGet 7
+                cbSuspend <- maybeGet 8
```

Comment:
For backwards compatibility, deserialization must fail if the suspend field is present for protocol versions that don't support suspending validators. (cf. how `supportDelegation` is used to guard deserialization of `ConfigureBaker`/`ConfiguredDelegation`.)

Reply:
I'll open a separate PR to add the necessary type level machinery that implements the guard.

Reply:
https://github.com/Concordium/concordium-base/pull/551